packages/turf-transform-rotate/test/in/polygon.geojson

```diff
@@ -0,0 +1,40 @@
+{
+  "type": "Feature",
+  "properties": {
+    "angle": 200,
+    "pivot": {
```

Comment:
Not too sure if `pivot` should be nested as a GeoJSON point, could simply be an Array `[2.48291015625, 27.819644755099446]`